include/mbedtls/md.h

```diff
@@ -49,6 +49,12 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160,
 } mbedtls_md_type_t;
 
+typedef enum {
+    MBEDTLS_SIG_RSA=0,
+    MBEDTLS_SIG_ECDSA,
+    MBEDTLS_SIG_NONE,
+} mbedtls_sig_type_t;
```

Comment:
By the way, a comment explaining why we need a new type instead of re-using `mbedtls_pk_type_t` could be useful, as it wasn't immediately obvious to me.